tools/db_bench_tool.cc

```diff
@@ -5220,7 +5220,7 @@ class Benchmark {
     }
 
     Duration duration(FLAGS_duration, reads_);
-    while (!duration.Done(1)) {
```

Comment:
So the previous duration is wrong if one batch has more than 1 entries?

Reply:
Yes, previously the benchmark would run longer than the user intended.